Fullscreen screenshots

DmitryBochkarev opened the pull request:
This feature will allow taking screenshots with the page that does not fit the current browser size.

Temporary repository of selenium-webdriver-crystal will be replaced with mainstream when [pull request ](https://github.com/ysbaddaden/selenium-webdriver-crystal/pull/29) accepted.

paulcsmith commented on shard.yml:
```diff
@@ -9,8 +9,11 @@ crystal: 0.25.0
-    github: paulcsmith/selenium-webdriver-crystal
-    tag: crystal-v0.25
+    github: DmitryBochkarev/selenium-webdriver-crystal
+    branch: windows-implementation-1
```
Looks like it was merged in. Could you point this to the official repo: https://github.com/ysbaddaden/selenium-webdriver-crystal now that it has crystal 0.25 support and your fullscreen changes?

paulcsmith commented on src/lucky_flow.cr:
```diff
@@ -36,12 +36,31 @@ class LuckyFlow
+    width = session.execute("return Math.max(document.body.scrollWidth, document.body.offsetWidth, document.documentElement.clientWidth, document.documentElement.scrollWidth, document.documentElement.offsetWidth);").as_i
+    height = session.execute("return Math.max(document.body.scrollHeight, document.body.offsetHeight, document.documentElement.clientHeight, document.documentElement.scrollHeight, document.documentElement.offsetHeight);").as_i
+    window = session.window
+    window.resize_to(width+100, height+100)
```
Could you run `crystal tool format spec src`. I think that will add spaces around the `+`

paulcsmith replied:
Also, why is the `+ 100` required?

DmitryBochkarev replied:
> Also, why is the + 100 required?

I don't remember roots of this; i guess it is a hack to ensure nothing left offscreen.

DmitryBochkarev commented:
@paulcsmith thank you for the review; I addressed your comments. There is issue shards take last released tag, I believe. Should I pin to commit or wait until a new tag is issued?

paulcsmith commented:
That looks great! I didn't think about the fact that it uses the latest tagged version. Let's keep this as-is for now and I'll hold off on merging it until @ysbadden makes a new release.

DmitryBochkarev commented:
I've pinned to master branch, and it is worked:)

paulcsmith commented:
I'll go ahead and merge this in and make a new version of my fork. Thanks for tackling this. This'll be super handy
